contracts/Fund.sol

```diff
@@ -133,6 +134,10 @@ contract Fund is Administratable {
       checkRecipient(recipient, orgFactoryContractAddress) == true,
       "Fund: Recipient contract was not created by the OrgFactory and is not allowed."
     );
+    require(
+      pendingGrants[grantId].recipient == address(0),
+      "Fund: Grant was already created."
```

Comment:
here I'm not allowing grants to be overwritten. If we remove `updateGrant` and remove this check, then we can update a grant by overwriting it here.

Reply:
I think preventing an overwrite is part of what OZ wanted to see from a validation/verification standpoint. That said, updating could come into play and is arguably part of the same previous feedback.  
```
In particular, allowed organizations cant be disallowed, grants can’t be modified nor deleted after being finalized, and organization claims can’t be modified.
```

Figuring `updateGrant` helps address the above.

Reply:
I also like the separate `updateGrant` function because that explicitness shows in event logs as well. Having a `GrantCreated` event emitted when you are updating a grant would be odd. So it seems we agree this functionality is useful, so I'll mark this discussion resolved for now